DTS-STN/secure-client-hub
# Update prod from staging to implement modal changes

### Description of proposed changes:
This PR updates the main branch to be in line with staging as of February 7th. The changes include all of the typescript updates to this point in addition to the removal of the modal popup on program links (on dashboard, profile, security settings and decisions review page).

### What to test for/How to test
1. Pull in branch
2. Type `npm run build` and `npm run start`
3. Confirm the application runs as expected

### Additional Notes
We need to look into a method of consolidating the changelog items from PRs/creating a changelog in our repo
